sdk/python/kfp/_client.py

```diff
@@ -139,6 +144,14 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client
     self._experiment_api = kfp_server_api.api.experiment_service_api.ExperimentServiceApi(api_client)
     self._pipelines_api = kfp_server_api.api.pipeline_service_api.PipelineServiceApi(api_client)
     self._upload_api = kfp_server_api.api.PipelineUploadServiceApi(api_client)
+    self._healthz_api = kfp_server_api.api.healthz_service_api.HealthzServiceApi(api_client)
```

Comment:
Are you sure this line will work with the latest stable version of `kfp_server_api`?
/cc @Bobgy @numerology

Reply:
Are people installing mismatching versions of `kfp_server_api` and the kfp client? I think that might be causing the error. If that is the case, maybe a solution would be to check the version of `kfp_server_api` before running this line?

Reply:
When installing packages, python does not upgrade dependencies if they are already installed and satisfy the requirements. So, unless the KFP package demands a newer version of `kfp_server_api`, it won't be updated.

Reply:
I thought the requirement was changed as it was already the recommendation.